src/accessory/thermostat-accessory.ts

```diff
@@ -39,6 +41,11 @@ export default class ThermostatAccessory extends BaseAccessory {
         this.device.displayName,
       );
 
+    this.service
+        .getCharacteristic(this.Characteristic.On)
+        .onGet(this.handlePowerGet.bind(this))
+        .onSet(this.handlePowerSet.bind(this));
+
```

Comment:
When starting homebridge, I get a warning due to this characteristic because homekit does not support `On` for thermostats. See [Thermostat Developer Docs](https://developers.homebridge.io/#/service/Thermostat).

Warning message:

```sh
 This plugin generated a warning from the characteristic 'On': Characteristic not in required or optional characteristic section for service Thermostat. Adding anyway.. See https://homebridge.io/w/JtMGR for more info.
```

Reply:
Indeed no need for Characteristic On, I removed it :)